src/handleApps.test.ts

```diff
@@ -1,7 +1,8 @@
-import { jest } from "@jest/globals";
+import { expect, jest } from "@jest/globals";
```

Comment:
`expect` is usually a global variable, did you have to import it for things to work? Asking just out of curiosity.